cfgov/unprocessed/js/molecules/Multiselect.js

```diff
@@ -352,7 +353,7 @@ function Multiselect( element ) { // eslint-disable-line max-statements, inline-
   function _bindEvents() {
     var inputs = _list.querySelectorAll( 'input' );
 
-    _bind( _search, {
+    domEvents.bindEvent( _search, {
```

Comment:
Sorry, I was confused. I had in my head this was binded for each item in the list, but it's a single time. It still may make sense to pull the functions out of the passed object literal so they're not anonymous, but this is fine for now.